src/Common/src/System/IO/PathInternal.Windows.cs

```diff
@@ -53,13 +53,20 @@ internal static partial class PathInternal
 
         internal static readonly char[] InvalidPathChars =
         {
-            '\"', '<', '>', '|', '\0',
```

Comment:
These characters can be used in file paths passed to Win32 functions like CreateFile?